integration-tests/locales/some/src/test/resources/application.properties

```diff
@@ -3,3 +3,4 @@ quarkus.locales=de,fr-FR,ja,uk-UA
 # used in your application properties. This test uses it only to verify compatibility.
 quarkus.native.user-language=cs
 quarkus.default-locale=en-US
+quarkus.test.arg-line=-Duser.language=de
```

Comment:
@zakkak Does it mean the user language  cs is ignored and should not be available in this case? Could you call https://github.com/quarkusio/quarkus/blob/main/integration-tests/locales/app/src/main/java/io/quarkus/locales/it/LocalesResource.java#L27 in > 24.2 test too then?

Reply:
AFAIK, it'll be available since `cs` is being added as an additional locale. The difference to Mandrel for JDK 23 is that you can no longer fix the default locale at build time. If you want a fixed locale the image needs `-Duser.language=cs` passed at runtime to fix it to `cs`.